app/presenters/concerns/document/metadata/rights.rb

```diff
@@ -31,8 +31,9 @@ def simple_rights(rights)
           }
         else
           license_flag_key = rights.template_license.present? ? rights.template_license : rights.id.to_s.upcase
-
           {
+            license_brief: rights.reusability ? t('global.facet.reusability.' + rights.reusability + '-brief') : false,
```

Comment:
This and the following call to `#t` should use the `scope` option, as with the call for `license_human`.